### PR TITLE
fix: fix relation config object and label manage

### PR DESCRIPTION
**Pull request by codding-y**

### CODE REVIEW

File: src/Services/Masa.Dcc.Service/Program.cs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the Apache License. See LICENSE.txt in the project root for license information.
 
+using Masa.Utils.Caching.Core.Models;
```
Comment (doddgu) [inline src/Services/Masa.Dcc.Service/Program.cs:4]: Move it to _Imports

File: src/Services/Masa.Dcc.Service/Services/OpenApiService.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
```
Comment (doddgu) [inline src/Services/Masa.Dcc.Service/Services/OpenApiService.cs:4]: Why you need mvc?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Services/Masa.Dcc.Service/Services/OpenApiService.cs (hunk 2 above)
Comment (codding-y) [reply src/Services/Masa.Dcc.Service/Services/OpenApiService.cs]: [FromBody] Attribute and so on is in  the namespace

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**